src/components/events/list-view/EventCard.tsx

```diff
@@ -9,15 +9,26 @@ interface EventCardProps {
   currentPage: number;
   filter?: string;
 }
+function checkNew(createdAt: Date) {
```

Comment:
Keep in mind that `createdAt` will be in UTC and `new Date()` will be in the local time zone. This might lead to a time zone difference.